templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_header.html.twig

```diff
@@ -38,6 +39,7 @@
                                         class="c-actionmenu__menuitem whitespace-normal"
                                         data-actionmenu-menuitem
                                         data-actionmenu-current
+                                        :data-cy="`baseHeader:{{ project.name | wysiwyg }}`"
```

Comment:
```suggestion
                                        :data-cy="`baseHeader:platformSwitcher:{{ index }}`"
```
We handle external links here. I think the idx is sufficient. And using a title leads to exposed configuration-details in the tests

Reply:
do you mean here `{{ loop.index }}`, right?
Done in https://github.com/demos-europe/demosplan-core/pull/3206/commits/7b4b0bf5699fbdc089c67e5d24699f7ee264100f